spec/factories/container_deployment_node.rb

```diff
@@ -0,0 +1,4 @@
+FactoryGirl.define do
+  factory :container_deployment_node, :class => "ContainerDeploymentNode" do
+  end
```

Comment:
Currently its not really needed for current tests, if we will need it later on ill add.